catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -209,6 +214,9 @@ QueryResponse query(
       boolean overrideFanoutRename,
       boolean fanoutEnabled)
       throws UnsupportedQueryException, FederationException {
+    DateFormat format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'");
+    format.setTimeZone(TimeZone.getTimeZone("UTC"));
+    String startTime = format.format(new Date());
```

Comment:
✏️ This would be more concise and wouldn't rely on string values if you changed these lines to:
`String startTime = (new Date()).toInstant().atOffset(ZoneOffset.UTC).format(DateTimeFormatter.ISO_DATE_TIME);`